src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50;
+  height: 60;
+}
+
+nav {
+  display: block;
+}
+
+ul {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  margin-left: 20px;
+  list-style: none;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+a {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
```

Comment:
move to html selector

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3647/files#r1280742001
didn't fix it